tests/logic.js

```diff
@@ -171,7 +171,7 @@ define([
                 var form = util.loadXML(MOTHER_REF_XML),
                     manager = form._logicManager;
                 assert.deepEqual(manager.caseReferences(),
-                    {load: {"/data/mug": ["parent/edd"]}});
+                    {load: {"/data/mug": ["#case/parent/edd"]}});
```

Comment:
Sorry, I don't understand what the problem is or what you're suggesting should be done differently. I think there must be some context I'm missing. As I understand it, `caseReferences()` returns (in the `load` value) a mapping of question paths to lists of referenced case properties, and this change makes case references less ambiguous.